source/fundamentals/atlas-search.txt

```diff
@@ -510,12 +510,36 @@ an ``establishedYear`` value greater than 1980 and less than 2020.
 
 The search returns the following results:
 
-.. code-block:: json
+.. code-block:: text
 
    { "_id" : 3, "make" : "PRS", "description" : "High-end guitars known for their quality.", "establishedYear" : 1985, "in_stock" : true, "rating" : 9 }
    { "_id" : 4, "make" : "Kiesel", "description" : "Quality guitars made only for custom orders.", "establishedYear" : 2015, "in_stock" : false, "rating" : null }
    { "_id" : 6, "make" : "Strandberg", "description" : "Modern guitars known for their headless models.", "establishedYear" : 1982, "in_stock" : false, "rating" : null }
 
+To search for documents in which the value of a specified field is within a
+range of strings, you must first create a :atlas:`token index
+</atlas-search/field-types/token-type/>` on the field. After you create the
+index, you can search for documents based on a range of strings as shown in the
+following example:
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs
+   :start-after: // start-range-string
+   :end-before: // end-range-string
+   :language: csharp
+   :dedent:
+
+The preceding example searches for any documents in which the string value of the
+``make`` field is greater than or equal to ``"Fender"`` and less than or equal
+to ``"Kiesel"``. The driver compares the string values in :wikipedia:`lexicographic order
+</Lexicographic_order>`. The search returns the following results:
```

Comment:
I: broken link i think it needs to be 
```suggestion
<Lexicographic_order>`. The search returns the following results:
```

Reply:
That gets me every time with these wiki links